Formula/ooniprobe.rb

```diff
@@ -200,41 +200,45 @@ def plist; <<-EOS.undent
    <dict>
      <key>Label</key>
        <string>#{plist_name}</string>
-     <key>Program</key>
-       <string>#{opt_bin}/ooniprobe</string>
+     <key>EnvironmentVariables</key>
+     <dict>
+       <key>PATH</key>
+       <string>#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+     </dict>
      <key>ProgramArguments</key>
      <array>
-       <string>-i</string>
-       <string>#{pkgshare}/current.deck</string>
+       <string>#{opt_bin}/ooniprobe-agent</string>
+       <string>run</string>
      </array>
      <key>RunAtLoad</key>
-       <false/>
+       <true/>
      <key>KeepAlive</key>
-       <false/>
+       <true/>
      <key>StandardErrorPath</key>
        <string>/dev/null</string>
      <key>StandardOutPath</key>
        <string>/dev/null</string>
-     <key>StartCalendarInterval</key>
-     <dict>
-       <key>Hour</key>
-       <integer>00</integer>
-       <key>Minute</key>
-       <integer>00</integer>
-     </dict>
+     <key>WorkingDirectory</key>
+       <string>#{var}/log/</string>
```

Comment:
Can you make this something specific to this formula e.g. `prefix` so it doesn't put random files in there? Thanks!